gapir/client/device_connection.go

```diff
@@ -233,13 +235,6 @@ func newHost(ctx context.Context, d bind.Device, abi *device.ABI, launchArgs []s
 	return &deviceConnectionInfo{port: port, authToken: authToken, cleanupFunc: cleanupFunc}, nil
 }
 
-var socketNames = map[device.Architecture]string{
```

Comment:
Is it safe to have one name for all sockets regardless of ISA? What do we lose?

Reply:
Good point, I try to explain that in the commit message: this socket name is used to decide on the file name for the unix socket, and this file is created on-device in a directory of the gapid APK, and we have a distinct APK per ABI. So the replayer for e.g. arm64 will listen on a unix socket backed by a file created under `/data/user/0/com.google.android.gapid.arm64v8a/`, and it will be a different directory for other ABIs (because they are different APKs).

The previous code was reflecting the ABI in the socket name, but that is redundant.

Reply:
Haha, sorry. I should have read that! LGTM